core/modules/ckeditor/ckeditor.module

```diff
@@ -777,3 +777,26 @@ function _ckeditor_theme_css($theme = NULL) {
   }
   return $css;
 }
+
+/**
+ * Implements hook_telemetry_info().
+ */
+function ckeditor_telemetry_info() {
+  // If this were in mymodule.module, "project" would assume to be "mymodule".
+  $info['cke_version'] = array(
+    'label' => t('CKeditor Version'),
+    'description' => t('The current version of CKEditor in use.'),
+    'project' => 'telemetry',
```

Comment:
The "project" key determines which project node on BackdropCMS.org receives the telemetry data. Assuming we want this to show on https://backdropcms.org/project/backdrop/telemetry, this should be `backdrop`

```suggestion
    'project' => 'backdrop',
```